### ISSUE TITLE
Introducing a New Public Preset for Achievements

### ISSUE THREAD

**Issue by remiroyc** [starknet-id/3sbt#3]
As discussed with @Th0rgal I think it could be interesting to create a public preset for SBT that doesn't require any whitelist system or password. The scenario I envision is a gaming service that wants to link achievements as SBTs on the Stark ID, where the SBT claim/transfer process could be initiated directly by the on-chain game contract.

Glad to have your input on this.

**Comment by fricoben** [comment]
Yep we're working on it. The main problem is as always `cairo 1`.
